pkg/apis/azureprovider/v1alpha1/types.go

```diff
@@ -0,0 +1,391 @@
+/*
+Copyright 2018 The Kubernetes Authors.
```

Comment:
```suggestion
Copyright 2019 The Kubernetes Authors.
```

New file. New year!!